rust_src/src/editfns.rs

```diff
@@ -247,7 +247,8 @@ pub fn insert_byte(byte: EmacsInt, count: Option<EmacsInt>, inherit: bool) {
 /// The optional third argument INHERIT, if non-nil, says to inherit text
 /// properties from adjoining text, if those properties are sticky.  If
 /// called interactively, INHERIT is t.
-#[lisp_fn(min = "1", intspec = "(list (read-char-by-name \"Insert character (Unicode name or hex): \") (prefix-numeric-value current-prefix-arg) t))")]
+#[lisp_fn(min = "1",
```

Comment:
Looks like the comma here isn't parsing. The docs suggest that multiline attributes are fine, so maybe the problem is that it's custom? Not sure, it's been a long time since I did any Rust coding.

Reply:
Yeah, my suspicion is our custom parsers are failing.

Reply:
Weird it's unique to OS X. It's also interesting to note that I get a warning on this exact line on Linux:

```
warning: static item is never used: `Sinsert_char`
   --> src/editfns.rs:250:1
    |
250 | / #[lisp_fn(min = "1",
251 | |           intspec = "(list (read-char-by-name \"Insert character (Unicode name or hex): \") (prefix-numeric-value current-prefix-arg) t))")]
    | |____________________________________________________________________________________________________________________________________________^
    |
    = note: #[warn(dead_code)] on by default
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

```

Reply:
I fixed this with `rustfmt_skip` but I forgot to add it after I rebased.